test/.eslintrc.json

```diff
@@ -1,13 +1,12 @@
 {
+	"root": true,
+
+	"extends": "../.eslintrc-browser.json",
+
 	"env": {
 		"browser": true
```

Comment:
> So we always want to reference global vars from `window` object?

I'm sure you meant it :) but just to be precise - only the browser-only globals; the one from the ECMAScript specification are used directly.

I'll update the PR.